examples/.joyride/scripts/ignore_form.cljs

```diff
@@ -21,7 +12,7 @@
     (aset editor "selection" original-selection)
     (p/do! (eu/insert-text!+ "#_" editor insert-position))))
 
-(when run-main?
+(when j/*file*
```

Comment:
Looks good to me. It didn't even occur to me that this would maybe already enough for the REPL! Except maybe in the `:load-file` nREPL operation where this variable would be bound:

https://nrepl.org/nrepl/ops.html#load-file

Reply:
>  supplied path and filename info to set source file and line number metadata

So that means this is not the whole solution to #4 then... Since we should actually be doing that, right?

Reply:
This is a part of the solution of #4 so let's continue. The other part is checking of we are currently in the invoked script. E.g. `(= *file* (joyride.core/invoked-script))` or so.